lib/src/signed_video_tlv.c

```diff
@@ -1024,7 +1024,7 @@ tlv_list_encode_or_get_size(signed_video_t *self,
       continue;
     }
 
-    if (tlv.is_always_present || self->has_recurrent_data) {
+    if (tlv.is_always_present || self->has_recurrent_data || self->is_golden_sei) {
```

Comment:
This is not the way to go. It is better if you set `self->has_recurrent_data = true;` in your new API. This recurrence stuff will be removed when we support golden SEI anyhow.